reflect.go

```diff
@@ -511,23 +533,20 @@ func (r *Reflector) reflectStructFields(st *Schema, definitions Definitions, t r
 			property = r.refOrReflectTypeToSchema(definitions, f.Type)
 		}
 
-		property.structKeywordsFromTags(f, st, name)
-		if property.Description == "" {
-			property.Description = r.lookupComment(t, f.Name)
+		unwrapped := property
```

Comment:
What is the objective here? I can't see where `unwrapped` is consumed, making this section much harder to read as I'd guess everything is happening on the pointer.

Reply:
https://github.com/invopop/jsonschema/pull/110/files#diff-672d1dd2b9193c8818108f625c29ff25feaea4c1c50cc36cfda18108cba3e1b2R540
We need this to work with pointer fields as well